include/seqan3/alphabet/structure/rna_structure_concept.hpp

```diff
@@ -100,7 +100,8 @@ concept bool rna_structure_concept = requires(structure_type val)
     { pseudoknot_id(val) } -> std::optional<uint8_t>;
 
     // this is delegated to a static class variable
-    { pseudoknot_support_v<structure_type> } -> uint8_t;
+    pseudoknot_support<std::remove_reference_t<structure_type>>::value;
+    pseudoknot_support_v<std::remove_reference_t<structure_type>>;
```

Comment:
change it here to 
```c++
requires pseudoknot_support_v<std::remove_reference_t<structure_type>>;
```

Reply:
As the value is not bool, I can test for
```cpp
requires pseudoknot_support_v<std::remove_reference_t<structure_type>> > 0;
```
Would this be better? Why do I need the `requires` here?